ontopy/utils.py

```diff
@@ -350,7 +350,7 @@ def load_uri(uri, dirname):
                 url = f"{baseuri}/{uri_as_str}"
             else:
                 url = os.path.join(baseuri if baseuri else dirname, uri_as_str)
-        url = normalise_url(url)
+        # url = normalise_url(url)
```

Comment:
Wouldn't this create other issues, since the URL is now not normalized?

Reply:
Both Jesper and I tried to see if we found such cases, but did not. The reason why it was commented out and not deleted completely was for easy reversal (and then searching for a new solution for Windows) if needed.